### ISSUE TITLE
Fix ugliness in examples imports

### ISSUE THREAD

**Issue by sah-huawei** [huawei-noah/SMARTS#1260]
In all of our example code, we have something like:
```python
if __name__ == "__main__":
    from argument_parser import default_argument_parser
else:            
    from .argument_parser import default_argument_parser
```

This was made necessary because our `aiohttp` dependency has an `examples` module too, which we addressed in PR #1120.

When we are able to update our dependency on `aiohttp` to a newer version (or remove it altogether) we should clean up the examples to go back to "normal" (non-relative) imports.

**Comment by Adaickalavan** [comment]
The latest version of `aiohttp` package does not cause issues with using the `example` module. The `argument_parser` imports have already been fixed in PR #1248 .

**Comment by Adaickalavan** [comment]
The latest version of `aiohttp` package does not cause issues with using the `example` module. The `argument_parser` imports can be fixed.
